src/components/starRating.tsx

```diff
@@ -0,0 +1,52 @@
+// ----- Imports ----- //
+
+import React, { ReactNode, ReactElement } from 'react';
+import { css } from '@emotion/core';
+import { brandAlt, text } from '@guardian/src-foundations/palette';
+
+import { Item, Design } from 'item';
+import { icons, spaceToRem } from 'styles';
+
+
+// ----- Subcomponents ----- //
+
+const starStyles = css`
+    ${icons}
+    background-color: ${brandAlt[400]};
```

Comment:
I think you could also go with `brandAltBackground.primary`, which is more semantic